docs/extending/block-patterns.md

```diff
@@ -1,4 +1,4 @@
-# Block patterns
+# Block Patterns
```

Comment:
Pretty sure our house style is "Sentence case" for top-level titles, but Yoli could confirm.